src/components/Notification/InactivityMessage.jsx

```diff
@@ -58,25 +61,54 @@ const InactivityMessage = () => {
     };
   }, []);
 
+  /* Starts a five minute timer when the inactivity window pops up. 
+     Logs the user out and refreshes the page if time elapses. */
+  useEffect(() => {
+    if (showPopup) {
+      logoutTimer.current = setInterval(() => {
+        if (seconds === 0 && minutes === 0) {
+          handleLogout();
+          window.location.reload(false);
+        } else if (seconds === 0) {
+          setSeconds(59);
+          setMinutes((minutes) => minutes - 1);
```

Comment:
For these set functions it looks like you're trying to use the previous state to set a new state.

If that's the case, I think you could use something like `setMinutes((prevMinute) => prevMinute - 1)` or `setMinutes((prev) => prev - 1)` instead it'll already imply what `minutes` would be.

Else, I believe something like `setMinutes(minute - 1)` should suffice.

Reply:
I tried the last thing you suggested, but the timer would stop counting down after one second. I ended up going with the ((prev) => prev - 1) you suggested.

Reply:
All good. Think it makes sense to use a previous state to update the next state for a timer.